source/aggregation-pipeline-builder.txt

```diff
@@ -172,6 +176,70 @@ your chosen view mode:
             
             |compass-short| returns your results in the document view. 
 
+   .. tab:: Stage Wizard 
+      :tabid: wizard 
+
+      .. procedure::
+         :style: normal
+         
+         .. step:: Select the :guilabel:`Stages` view.
+            
+            In the aggregation pipeline pane, ensure the 
+            :guilabel:`{} Stages` toggle switch is selected.
+
+            .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
+               :figwidth: 300 px
+               :alt: Toggle on stage view mode 
+
+         .. step:: Open the :guilabel:`Stage Wizard` card. 
+            
+            To the right of the view mode toggle, click the funnel icon to open 
```

Comment:
we are replacing this icon with `Wizard` icon that was recently added in Leafygreen. Should we update this PR with new screenshorts and text change?

Reply:
Do you know when those UI changes would be pushed out to the Compass beta? In the case that it won't be out for a bit, this PR pushes to a feature branch, so we can proceed with this PR (once I address your other comments) and update the screenshots and text in a separate ticket once that's pushed out to Compass beta for testing.

Reply:
sgtm.